src/internal.h

```diff
@@ -356,14 +356,10 @@ DISPATCH_EXPORT DISPATCH_NOTHROW void dispatch_atfork_child(void);
 		do { (elm)->field.tqe_prev = NULL; } while (0)
 
 #if DISPATCH_DEBUG
-// sys/queue.h debugging
-#if defined(__linux__)
-#define QUEUE_MACRO_DEBUG 1
```

Comment:
this breaks people using libbsd-dev and doesn't hurt, please do not remove.
what you probably want to do is to both set QUEUE_MACRO_DEBUG and TRASHIT to support both (?)